rules-reviewed/camel3/camel2/tests/data/xml-renamed-components/pom.xml

```
@@ -23,6 +23,54 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-mongodb3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-hdfs2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-hdfs</artifactId>
+        </dependency>
```

added test case to avoid the "chain of changes" 
```suggestion
                    <version>2.24.3</version>
        </dependency>
        <dependency>
            <groupId>org.apache.camel</groupId>
            <artifactId>camel-hdfs</artifactId>
            <version>3.0.0</version>
        </dependency>
```